test/evergreen.yml

```diff
@@ -7346,4 +7346,41 @@ buildvariants:
     additional_san_vars: export ASAN_OPTIONS="$COMMON_SAN_OPTIONS:unmap_shadow_on_exit=1"
   tasks:
     - name: compile
-    - name: ".stress-test-disagg-asan-fail"
+    - name: ".stress-test-disagg-san-fail"
+    - name: checkpoint-test
+
+- name: amazon2023-disagg-msan-stress
```

Comment:
nit- can we add failure expected here too, I noticed this makes it easier in BBUI to detect these.

Reply:
The display name has "[Failure Expected] (ARMV9) Amazon 2023 MSAN Disagg Stress tests" already. Is that what you are talking bout?